guides/common/modules/con_redis-tuning.adoc

```diff
@@ -5,7 +5,12 @@
 
 Redis is an in-memory data store.
 It is used by multiple services in {Project}.
+ifdef::foreman-el,foreman-deb[]
+The Dynflow tasking system uses it to track its tasks.
+endif::[]
+ifdef::katello,orcharhino,satellite[]
 The Dynflow and Pulp tasking systems use it to track their tasks.
```

Comment:
Pulp's "new" tasking system uses PostgreSQL, not Redis. https://projects.theforeman.org/issues/32891 made that the default, so Foreman 2.5. https://github.com/theforeman/puppet-pulpcore/commit/6f09c2ff328fac9fcaf1388d9112a5130db3e2d7 is where we dropped the old one, which landed in Foreman 3.3.

That reminds me: since Foreman 3.9 the default cache is also Redis. https://docs.theforeman.org/3.9/Release_Notes/index-katello.html#_redis_is_the_default_cache_type